code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -145,6 +145,54 @@
 		GLOB.pre_setup_antags -= changeling
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//              ELDRITCH CULT               //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/ecult
```

Comment:
Could you change the instances of `ecult` for something like `e_cult` to make the difference more visible at a glance?